xtask/src/examples.rs

```diff
@@ -363,10 +360,7 @@ pub fn build_functions_example(opt: &RunFunctionsExamples, scope: &Scope) -> Ste
             functions_example.construct_application_build_steps(),
             // Build the server first so that when running it in the next step it will start up
```

Comment:
Can you clarify in a comment here or elsewhere that we always build all server variants, even if the particular example being built / run only needs one?

Reply:
Done (on `build_functions_server_variants`).